install.py

```diff
@@ -356,10 +350,18 @@ def validate_path(path):
     cmake_flags += ["-Dlegate_core_ROOT=%s" % legate_dir]
 
     cmake_flags += extra_flags
+    ninja_flags = [f"-j{str(thread_count)}"]
+    if verbose:
+        if cmake_generator == "Unix Makefiles":
+            ninja_flags += ["VERBOSE=1"]
+        else:
```

Comment:
Should we call these `generator_flags` since they are relevant to both ninja and make?

Reply:
sure, changed it to `build_flags`